cpp/arcticdb/storage/library_manager.cpp

```diff
@@ -125,7 +125,7 @@ void LibraryManager::remove_library_config(const LibraryPath& path) const {
     store_->remove_key(RefKey{StreamId(path.to_delim_path()), entity::KeyType::LIBRARY_CONFIG}).wait();
 }
 
-std::shared_ptr<Library> LibraryManager::get_library(const LibraryPath& path, const StorageOverride& storage_override) {
+std::shared_ptr<Library> LibraryManager::get_library(const LibraryPath& path, const StorageOverride& storage_override, const StorageCredential& storage_credential) { //Currently only support one storage per library with external API
```

Comment:
Bit worried about the `one storage per library with external API` assumption bearing in mind the upcoming replication stuff. Would it make sense to put the `StorageCredential` stuff on the existing `StorageOverride` object?